lightning/src/routing/network_graph.rs

```diff
@@ -375,17 +375,20 @@ impl<C: Deref , L: Deref > RoutingMessageHandler for NetGraphMsgHandler<C, L> wh
 		let batch_count = batches.len();
 		for (batch_index, batch) in batches.into_iter().enumerate() {
 			// Per spec, the initial first_blocknum needs to be <= the query's first_blocknum and subsequent
-			// must be >= the prior reply. We'll simplify this by using zero since its still spec compliant and
-			// sequence completion is now explicitly.
-			let first_blocknum = 0;
+			// must be >= the prior reply. We'll simplify this by using the query's first_blocknum since its
```

Comment:
nit: s/and subsequent must be >= the prior reply/the subsequent first_blocknums must be >= the prior reply's first_blocknum ?